helm/slurm-operator/values.yaml

```diff
@@ -34,7 +34,7 @@ controllerManager:
       isPrometheusCrdInstalled: "false"
       slurmOperatorWatchNamespaces: '*'
     image:
-      repository: cr.nemax.nebius.cloud/crnlu9nio77sg3p8n5bi/slurm-operator
+      repository: ghcr.io/nebius/soperator/slurm-operator
```

Comment:
Do we want to use `soperator/slurm-operator` name instead of `soperator/soperator`?

Reply:
That's a lot of changes (((

Reply:
ok